components/layouts/blog/blog-homepage-section.hbs

```diff
@@ -163,6 +163,9 @@
             </div><!-- .content-slider__cards-list .grid-col-content__four--homepage-blog -->
 
         </div><!-- .content-slider-blog-->
+        <div class="content-block--homepage-blog--blog-button">
+            <a href="/blog/" class="button--medium-primary">View All</a>
```

Comment:
Double check this. I think it should be button--large-primary

Reply:
Correct and fixed.